testutils/src/main/java/com/microsoft/identity/internal/testutils/mocks/MockServerResponse.java

```diff
@@ -43,6 +43,20 @@ public static HttpResponse getMockTokenSuccessResponse() {
         return new HttpResponse(200, mockResponse, new HashMap<String, List<String>>());
     }
 
+    public static HttpResponse getMockTokenSuccessResponse(final String localAccountId, final String tenant, final String issuer, final String rawClientInfo, final String accessToken) {
+        final String mockResponse = "{\n" +
```

Comment:
Rather than strconcat this data together, can we use `JSONObject` or equivalent to construct the data?